ci/bootstrap.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: infrastructure
+value: 1000000
+globalDefault: false
+description: "For infrastructure like gateway."
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: production
+value: 900000
+globalDefault: false
+description: "For production services like apiserver, batch, etc."
+---
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: user
+value: 500000
```

Comment:
The relative value of these classes doesn't matter, right? I feel like this sets a bad precedent of seemingly arbitrary choices. Should we maybe do factors of two? So, 1M, 500k, 250k, 0?

Reply:
FYI, this code isn't new, it came from vdc/ stuff from a long time ago.  So I'd probably rather not change it all things being equal.  K8s priority classes are 32-bit, and user classes should be in the range 0-1B.  Numbers greater than 1B are reserved for system processes that shouldn't be evicted/preempted.

Yes, the numbers are relatively.  All the examples I've seen use big numbers like this, so that's what I did.  I think the point is to leave space if you want to insert other priorities (so don't make them 1, 2, 3, etc.)